pkgs/applications/editors/heh/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "heh";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "ndd7xv";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-IIF/bkTLwR8pCs/hJ625T3NsiKf/6Zf1cW2i4lsiK4U=";
+  };
+
+  cargoHash = "sha256-tDvqaNVuzv1BlS/oNI1D/WV1b5uHreT3Ak/6ruqKXQc=";
+
+  meta = with lib; {
+    description = "A cross-platform terminal UI used for modifying file data in hex or ASCII.";
+    homepage = "https://github.com/${src.owner}/${src.repo}";
```

Comment:
Please use the raw URL, so it can easily be opened. The `rev` is fine for the `changelog`.

Reply:
Fixed :+1: